crypto/fipsmodule/cpucap/cpu_intel.c

```diff
@@ -114,7 +114,9 @@ static uint64_t OPENSSL_xgetbv(uint32_t xcr) {
   return (uint64_t)_xgetbv(xcr);
 #else
   uint32_t eax, edx;
-  __asm__ volatile ("xgetbv" : "=a"(eax), "=d"(edx) : "c"(xcr));
+  // Some old assemblers don't support the xgetbv instruction so we emit
+  // the opcode of xgetbv directly.
+  __asm__ volatile (".byte 0x0f, 0x01, 0xd0" : "=a"(eax), "=d"(edx) : "c"(xcr));
```

Comment:
Is the related reference available? 
`xgetbv` to `".byte 0x0f, 0x01, 0xd0"`

Is this transfer always guaranteed(by what)? I think it's safer to enable this transfer only when `MY_ASSEMBLER_IS_TOO_OLD_FOR_AVX` is defined.